src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarReader.cs

```diff
@@ -322,19 +305,15 @@ private bool TryProcessExtendedAttributesHeader(TarHeader firstHeader, bool copy
                 throw new FormatException(SR.TarTooManyGlobalExtendedAttributesEntries);
             }
 
-            // Can't have two metadata entries in a row, no matter the archive format
+            // Can't have two extended attribute metadata entries in a row
             if (secondHeader._typeFlag is TarEntryType.ExtendedAttributes)
             {
                 throw new FormatException(string.Format(SR.TarUnexpectedMetadataEntry, TarEntryType.ExtendedAttributes, TarEntryType.ExtendedAttributes));
```

Comment:
Do the `tar` or `pax` commands also bail out in this case?

Reply:
This question, regarding whether it's actually illegal to have two sequential PAX GEA records, does not seem to have been answered.

Reply:
Just tested this by manually editing an existing tar file with HxD and inserting extra metadata entries before a normal entry. I discovered two behaviors:

- If there are two or more extended headers in a row, and they have different extended attributes, the `tar` tool fails with the error message: "tar: Malformed extended header. Missing newline. Exiting with failure status due to previous errors".
- If there are two or more extended headers in a row and they are identical, the `tar` tool succeeds, no matter how many identicaly extended attributes headers are found.

I could address this case, but I would prefer to do it in a separate PR. It's a super rare edge case scenario.